doozerlib/cli/cli_opts.py

```diff
@@ -44,6 +44,10 @@ def global_opt_default_string():
         'env': 'DOOZER_RHPKG_CONFIG',
         'help': 'Config file path for rhpkg calls'
     },
+    'registry_config': {
+        'env': 'DOCKER_CONFIG',
+        'help': 'Config path for the image registry'
```

Comment:
there's a complication here. `DOCKER_CONFIG` is a directory (see https://docs.docker.com/engine/reference/commandline/cli/#change-the-docker-directory). `oc --registry-config` is the `config.json` inside that directory (`DOCKER_CONFIG` may have other contents relevant to docker).

`DOCKER_CONFIG=./foo doozer images:push` should work the same as `doozer --registry-config=./foo/config.json images:push`.